pkg/logql/log/fmt.go

```diff
@@ -130,6 +133,35 @@ func addLineAndTimestampFunctions(currLine func() string, currTimestamp func() i
 	return functions
 }
 
+// toEpoch converts a string with Unix time to an time Value
+func toEpoch(epoch string) (int64, error) {
```

Comment:
not sure why this returns an int64 when the others return a time.Time?

Reply:
This code was replaced by 1 function.